rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/indices.open/10_basic.yml

```diff
@@ -8,7 +8,7 @@
         index: test_index
         body:
           settings:
-            number_of_replicas: 0
+            auto_expand_replicas: 0-1
```

Comment:
`':qa:core-rest-test` runs with a single node. 
This should allow us to have a green cluster when running `':qa:core-rest-test` as well as have a search/replica shards when running in ingest/search mode.

Reply:
I took a slightly different route in https://github.com/elastic/elasticsearch/pull/93386 and #109, where a global index template is defined to override the number of replica in stateless tests. 

I failed to have `auto_expand_replicas` work reliably: the index becomes green and then the number of replicas got updated in another cluster state update, causing the test failing.